src/WebexSDKAdapter.js

```diff
@@ -0,0 +1,13 @@
+export default class WebexSDKAdapter {
+  /**
+   * Creates a new instance of the WebexAdapter.
+   * The primary datasource model comes from the following structure:
+   * WebexSDKAdapter = {}
+   *
+   * @param {Object} datasource The primary datasource the json adapter will be using.
```

Comment:
I think we can call this `sdk` since we know the datasource for this adapter is the SDK.
Make sure to update the description accordingly